services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -73,6 +73,14 @@ module.exports = {
   Content: {
     __resolveType: resolveType,
 
+    shortName: (content) => {
+      const shortName = get(content, 'shortName', '').trim();
+      const mutatedName = get(content, 'mutations.Website.name', '').trim();
+      if (mutatedName) return mutatedName;
+      if (shortName) return shortName;
```

Comment:
Shouldn't short name be returned before the website revision if specified?

Reply:
Nope, other way around

Reply:
I kid